src/structurizr/view/view.py

```diff
@@ -46,32 +46,29 @@ class ViewIO(BaseModel, ABC):
 
     key: str
     description: str
-    software_system_id: str = Field(default="", alias="softwareSystemId")
+    software_system_id: str = Field(default=None, alias="softwareSystemId")
```

Comment:
We're being somewhat inconsistent with some strings defaulting to empty string and others to `None`. What do you use as guidance?

```suggestion
    software_system_id: Optional[str] = Field(default=None, alias="softwareSystemId")
```